client/state/data-layer/wpcom/concierge/schedules/initial/from-api.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import makeJsonSchemaParser from 'lib/make-json-schema-parser';
+import responseSchema from './schema';
+import { transform as appointmentTransformer } from 'state/data-layer/wpcom/concierge/schedules/appointments/detail/from-api';
```

Comment:
Nice job seeing this and re-using it.

Reply:
Isn't is possible the schema to also be reused?